### ISSUE TITLE
Review this P2P code for creating event core (related project)

### ISSUE THREAD

**Issue by mobb** [EDIorg/ecocomDP#100]
Received this link from Enrique Montes. His P2P project has created templates for generating event core from data templated in excel (confirm).  
https://github.com/diodon/P2P-templates

look at these and compare to our workflows.

**Comment by clnsmth** [comment]
@mobb I don't think we need to pursue this since we have `convert_to_dwca()`. May I close this issue?
